operations/kubernetes/Kyverno.md

```diff
@@ -323,6 +323,75 @@ Also, it generates the report of policy violation in respective workloads. The f
 
     If there are any discrepancies, look at the detailed policy report to triage the issue.
 
+## What is new in the HPE CSM 1.4 release
```

Comment:
This is targetting the `main` branch, which is CSM-1.5, but this mentions CSM-1.4.  Is this correct?

Reply:
Changed the information to suit for releases after 1.4.